http_logs/challenges/default.json

```diff
@@ -140,14 +148,21 @@
       "name": "append-index-only-with-ingest-pipeline",
       "description": "Indexes the whole document corpus using Elasticsearch default settings. We only adjust the number of replicas as we benchmark a single node cluster and Rally will only start the benchmark if the cluster turns green. Document ids are unique so all index operations are append only. Runs the documents through an ingest node pipeline to parse the http logs. May require --elasticsearch-plugins='ingest-geoip' ",
       "schedule": [
+        {{ rally.collect(parts="common/setup-schedule.json") }},
         {
-          "operation": "delete-index"
-        },
-        {
+          {%- if index_mode == "logsdb" or index_type == "data_stream" %}
+            {%- set indexing_operation_type = "create-data-stream" %}
+          {%- endif %}
           "operation": {
-            "operation-type": "create-index",
-            "settings": {{index_settings | default({}) | tojson}}
-          }
+            "operation-type": {{ indexing_operation_type | default("create-index") | tojson }},
+            "settings": {%- if index_settings is defined %} {{index_settings | tojson}} {%- else %} {
+              "index.sort.field": "@timestamp",
```

Comment:
Looking at the template, not sure what else can be a good sorting field without knowing the data, I think just sorting by timestamp is good.

Reply:
There is basically nothing there `@timestamp`, `message`, `clientip`, `request`, `status`, `size` and `geoip`. At the end it depends on queries...the idea is that we want to sort to favor query latency.

Reply:
As a result I think we can use this as an example to udnerstand what happens if `host.name` is missing. I would just remove the index sorting configuration and rely on defaults. In practice since `host.name` is missing it would result in just sorting on `@timestamp` (the `host.name` field is injected and will be empty).